tests/console/postgresql_server.pm

```diff
@@ -27,8 +27,8 @@ sub run {
     # start the postgresql service
     assert_script_run 'systemctl start postgresql.service', 200;
 
-    # check the status
-    assert_script_run 'systemctl show -p ActiveState postgresql.service | grep ActiveState=active';
+    # check the status, if failed, get possible cause
+    assert_script_run 'systemctl is-active postgresql.service || systemctl status postgresql.service';
```

Comment:
Hm, I would be careful with that. It could confuse the reviewers when the test is not actually failing in this line when the service is not active. How about a custom `post_fail_hook` instead? This would make it more explicit that the job failed but we are still trying to collect logs and help investigation.